tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1288,6 +1297,9 @@ def build(self, input_fn):
     if not input_fn:
       raise RuntimeError("input_fn is None. Method build() needs input_fn "
                          "to be specified in order to build TensorRT engines")
+    if self._need_calibration and self._need_trt_profiles() and \
+        self._calibration_input_fn == None:
+        raise RuntimeError("Need to call convert() first")
```

Comment:
Maybe we should make it clear to call convert with a calibration_input_fn in the error message?